tox.ini

```
@@ -20,6 +20,14 @@ commands =
    poetry install -v {env:INSTALL_OPTS}
    poetry run pytest --quiet --cov=generated/nidaqmx --cov-append --cov-report= --junitxml=test_results/system-{envname}.xml {env:PYTEST_OPTS} {posargs}
```

Let's log the library name and encoding. This will let us verify it's testing the right DLL and help us debug why i18n test cases are/aren't running.
```suggestion
   poetry install -v {env:INSTALL_OPTS}
   poetry run python -c "from nidaqmx._lib import lib_importer; print(f'Library: {lib_importer.windll._library._name}\nLibrary encoding: {lib_importer.encoding}')"
   poetry run pytest --quiet --cov=generated/nidaqmx --cov-append --cov-report= --junitxml=test_results/system-{envname}.xml {env:PYTEST_OPTS} {posargs}
```